src/components/RenderAfterMount/RenderAfterMount.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {useMounted} from '../../utilities/use-mounted';
+
+interface Props {
+  children?: React.ReactNode;
+}
+
+export function RenderAfterMount({children}: Props) {
```

Comment:
Per comment below, feedback depends on what this is for. I wonder if there is a name that more accurately describes the intent more succinctly than the `how`?

Prefixing with `Render` feels a bit superflous, perhaps `AfterMount` (if this is for after *every* mount), 
`AfterInitialMount` (if this is for after the initial mount)

What do you think to accepting a `fallback` prop (defaulting to null) ([like Suspense](https://itnext.io/what-the-heck-is-this-in-react-suspense-c5e641e487a) to allow you to define initial content.

Reply:
> What do you think to accepting a fallback prop (defaulting to null) (like Suspense to allow you to define initial content.

I like the idea 👍 This is for rendering content after mount, so on the client. For example, cases when we have to use `useLayoutEffect` instead of `useEffect`.